cm/resource.go

```diff
@@ -3,6 +3,13 @@ package cm
 // TODO: should there be separate types for owned vs borrowed handles?
 type Resource uint32
 
+// "Upon initialization, table element 0 is allocated and set to None,
```

Comment:
This should follow the godoc convention of `ResourceNone is ...` in the doc header, and use the `[...]` links consistent with the other doc comments in this package.